week08_pomdp/practice_pytorch.ipynb

```diff
@@ -6,17 +6,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import sys\n",
-    "if 'google.colab' in sys.modules:\n",
-    "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/0ccb0673965dd650d9b284e1ec90c2bfd82c8a94/week08_pomdp/atari_util.py\n",
-    "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/0ccb0673965dd650d9b284e1ec90c2bfd82c8a94/week08_pomdp/env_pool.py\n",
-    "\n",
-    "# If you are running on a server, launch xvfb to record game videos\n",
-    "# Please make sure you have xvfb installed\n",
-    "import os\n",
-    "if type(os.environ.get(\"DISPLAY\")) is not str or len(os.environ.get(\"DISPLAY\")) == 0:\n",
-    "    !bash ../xvfb start\n",
-    "    os.environ['DISPLAY'] = ':1'"
+     "import sys\n",
+     "if 'google.colab' in sys.modules:\n",
+     "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/0ccb0673965dd650d9b284e1ec90c2bfd82c8a94/week08_pomdp/atari_util.py\n",
+     "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/0ccb0673965dd650d9b284e1ec90c2bfd82c8a94/week08_pomdp/env_pool.py\n",
+     "    # Setup the attari driver for video games\n",
```

Comment:
This is not quite correct as there is no "driver" involved. Since this script is used in some other notebooks as well, I think it's fine to just remove this comment entirely.

Reply:
![image](https://user-images.githubusercontent.com/72295771/201537388-6ee2d3d9-32b0-4636-acb2-58b1a305fd43.png)

Without this part, we can't download `ROM.rar` or `xvfb`, meaning no video will be recorded or work on the notebook!